sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MethodOverloadsShouldBeGrouped.cs

```diff
@@ -274,4 +279,68 @@ protected override void DoSomething(bool b)
 
     }
 
+    public class InterfaceImplementationTogether : IEqualityComparer<int>, IEqualityComparer<string>
```

Comment:
I think it would be great to add a few additional tests with:
- a class which implements interfaces with at least a common method (e.g IFirst with M1() and M2() and ISecond with M2() and M3())
- a class which implements interfaces with default members

Reply:
What should it prove?
Implicit implementation will create method `M1`, `M2` and `M3`. Nothing interesting to raise.
Explicit implementation will be ignored. Ignoring explicit interfaces is documented in the test on line 117.
C#8 default member implementation? That should not interact with this rule. It will either not be implemented => not in syntax => does nothing. Or be implemented as interface and that's already tested.

Reply:
I was thinking at this case:
```
    public interface ISecond
    {
        void A();
        void Cancel();
        void B();
    }

    public class ImplementsBoth : ICancel, ISecond
    {
        public void Cancel(bool b) { }
        public void Renew() { }

        public void A() { }
        public void Cancel() { } // Noncompliant
        public void B() { }
    }
```

Reply:
That's a good catch. It did not behave as expected.